translate/lang/el.py

```diff
@@ -43,19 +45,19 @@ class el(common.Common):
         """ % sentenceend, re.VERBOSE | re.UNICODE)
 
     puncdict = {
-        u"?": u";",
-        u";": u"·",
+        "?": ";",
+        ";": "·",
     }
 
     # Valid latin characters for use as accelerators
-    valid_latin_accel = u"abcdefghijklmnopqrstuvwxyz" + \
-                        u"ABCDEFGHIJKLMNOPQRSTUVWXYZ" + \
-                        u"1234567890"
+    valid_latin_accel = "abcdefghijklmnopqrstuvwxyz" + \
+                        "ABCDEFGHIJKLMNOPQRSTUVWXYZ" + \
+                        "1234567890"
 
     # Valid greek characters for use as accelerators (accented characters
     # and "ς" omitted)
-    valid_greek_accel = u"αβγδεζηθικλμνξοπρστυφχψω" + \
-                        u"ΑΒΓΔΕΖΗΘΙΚΛΜΝΞΟΠΡΣΤΥΦΧΨΩ"
+    valid_greek_accel = "αβγδεζηθικλμνξοπρστυφχψω" + \
+                        "ΑΒΓΔΕΖΗΘΙΚΛΜΝΞΟΠΡΣΤΥΦΧΨΩ"
```

Comment:
Since these lines are being touched I suggest using the following in order to drop string concatenation and `\` usage:

```python
    valid_greek_accel = ("αβγδεζηθικλμνξοπρστυφχψω"
                         "ΑΒΓΔΕΖΗΘΙΚΛΜΝΞΟΠΡΣΤΥΦΧΨΩ")
```

The same above.